mayu.service

```diff
@@ -1,15 +1,37 @@
 [Unit]
-Description=Mayu
-After=docker.service
-Requires=docker.service
+Description=mayu
+Requires=etcd.service
 
 [Service]
-Restart=always
+Restart=on-failure
+RestartSec=1
+StartLimitInterval=300s
+StartLimitBurst=3
 TimeoutStartSec=0
-ExecStart=/usr/bin/docker run --rm --cap-add=NET_ADMIN --net=host \
+Environment="IMAGE=quay.io/giantswarm/mayu:c832dd12fa8f5cc77101ae05820106ace18504f7"
```

Comment:
For my interest, why is this unit file in here? I feel like this should just be an example file with some placeholders. I can imagine people will just run it and wonder why there is a specific version being used and not something like the latest.